atomicapp/constants.py

```diff
@@ -47,9 +47,11 @@
 ANSWERS_FILE_SAMPLE = "answers.conf.sample"
 ANSWERS_FILE_SAMPLE_FORMAT = 'ini'
 WORKDIR = ".workdir"
-LOCK_FILE = "/run/lock/atomicapp.lock"
 HOST_DIR = "/host"
 
+# Use /tmp/ so that we can grab the lock as non-root
+LOCK_FILE = "/tmp/atomicapp.lock"
```

Comment:
Did some more thinking about this. I have to change it back and solve it a different way. In the past multiple instances of atomicapp running on a system share /run/lock on the host because /run is mounted into the container. So they would be competing for the same lockfile. 

In this case they would not because /tmp from the host isn't mounted in the container.